.github/workflows/CI.yml

```diff
@@ -0,0 +1,193 @@
+name: CI
+on:
+  # always execute docker build when something is pushed to main/master or release-* branches
+  push:
+    branches:
+      - "master"
+      - "main"
+      - "release-*"
+  # in addition, execute for pull requests to those branches
+  pull_request:
+    branches:
+      - "master"
+      - "main"
+      - "release-*"
+
+  workflow_dispatch: # run integration tests only when triggered manually
+
+defaults:
+  run:
+    shell: bash
+jobs:
+  prepare_ci_run:
+    name: Prepare CI Run
+    # Prepare CI Run looks at what has been changed in this commit/PR/... and determines which artifacts should be
+    # built afterwards (in other jobs that depend on this one).
+    runs-on: ubuntu-20.04
+    outputs: # declare what this job outputs (so it can be re-used for other jobs)
+      # build config
+      # metadata
+      GIT_SHA: ${{ steps.extract_branch.outputs.GIT_SHA }}
+      BRANCH: ${{ steps.extract_branch.outputs.BRANCH }}
+      BRANCH_SLUG: ${{ steps.extract_branch.outputs.BRANCH_SLUG }}
+      VERSION: ${{ steps.get_version.outputs.VERSION }}
+      DATE: ${{ steps.get_datetime.outputs.DATE }}
+      TIME: ${{ steps.get_datetime.outputs.TIME }}
+      DATETIME: ${{ steps.get_datetime.outputs.DATETIME }}
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3.0.2
+        with:
+          fetch-depth: 0 # need to checkout "all commits" for certain features to work (e.g., get all changed files)
+
+      - name: Load CI Environment from .ci_env
+        id: load_ci_env
+        uses: c-py/action-dotenv-to-setenv@v3
+        with:
+          env-file: .ci_env
```

Comment:
do you plan to have multiple values in the `.ci_env` file in the future?

Reply:
In keptn-contrib/sandbox we rely on information from .ci_env for "global environment" variables, e.g., the container image name. We used to have additional information in there, but it started to get less when we moved to ghcr.io instead of docker.io.
We can discuss ways to migrate away from this file, if you have any. But I'd like to avoid having `IMAGE=helm-service` in 20 different places.